mender-deb-package

```diff
@@ -21,25 +21,56 @@ verify_output_directory_exists() {
   fi
 }
 
-extract_client_version() {
+verify_script_arguments() {
+  if [ $# -ne 3 ]; then
+    show_help_and_exit
+  fi
+  DEB_PACKAGE=$1
+  REPO_URL=$2
+  VERSION=$3
+}
+
+checkout_repo() {
+  go get -d $REPO_URL
+  cd /root/go/src/$REPO_URL
+  git checkout $VERSION
+}
+
+get_package_version() {
```

Comment:
Would consider using `get_deb_version` to match the variable name it's storing it in.